### ISSUE TITLE
[Designer Module 7: Forms Design] Typo in the list of applied expertise for Instructors

### ISSUE THREAD

**Issue by carlosapaduarte** [w3c/wai-curricula#423]
The fifth item reads "WCAG Success Criteria", but it should be "WCAG Success Criterion"

**Comment by daniel-montalvo** [comment]
Thanks for pointing this out. @carlosapaduarte 
I have gone through the resource and there was one more instance of this wrong plural.

Fixed. Will show up when updating preview.
